src/screens/Drawer/CustomDrawer.tsx

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import { SafeAreaView, Text, View, Image, StyleSheet, TouchableOpacity } from "react-native";
+import {
+    DrawerContentComponentProps,
+    DrawerContentScrollView,
+    DrawerItemList,
+  } from '@react-navigation/drawer';
+import { LinearGradient } from 'expo-linear-gradient';
+import { SIZES, COLORS, FONTS } from '../../styles/theme';
+import { AntDesign } from "@expo/vector-icons";
+import * as Progress from 'react-native-progress';
+  
+
+export default function CustomDrawerContent(props: DrawerContentComponentProps) {
+
+    const loading = false;
+    const kshBalance = 567.37;
+    const cUSDBalance = 5.67;
+    const loadingMessage = "Loading..."
+    
+
+    return (
+      <SafeAreaView style={styles.container}>
+        <View>
+            <LinearGradient
+                colors={[
+                    "rgba(255, 140, 161, 0.08)",
+                    "rgba(252, 207, 47, 0.08)",
+                    "rgba(255, 255, 255, 0.08)",
+                    "rgba(248, 48, 180, 0.08)",
+                    "rgba(47, 68, 252, 0.08)",
+                ]}
+                start={[0, 1]}
+                end={[1, 0]}
+                style={styles.header}
+            >
+                <View style={{ marginBottom: 80, marginTop: 31}} >
+                    <Image 
+                      source={require("../../assets/images/drawer/dummyimages/dummy_identicone.png")}
+                      style={styles.identiconImg}
+                    ></Image>
+
+                    <View style={{ flexDirection: "row", height: 'auto' }}>
+                        <Image 
+                          source={require("../../assets/images/drawer/dummyimages/kenya_flag.png")}
+                          style={styles.phoneCountryFlag}
+                        />
+                        
+                        <Text style={styles.phoneNumberTest}>+254 706 427718</Text> 
+                       
+                    </View>
+                </View>
+
+                <TouchableOpacity onPress={() => props.navigation.closeDrawer()} style={{ marginTop: 20 }}>
+                    <AntDesign name="close" size={24} color="black" />
```

Comment:
@Allen-Muhani @Chegemartinx @AlphonceMutebi there is styling here. how can we centralize that?

Reply:
It is specific to that component. So I am of the opinion that it stays as is.

Reply:
@steffenkrogmann  seems ok ....to me